data/metric/metrics.go

```diff
@@ -59,20 +52,12 @@ func NewSet(eventType string, storer persist.Storer, attributes ...Attribute) (s
 }
 
 // AddCustomAttributes add customAttributes to MetricSet
-func AddCustomAttributes(metricSet *Set, customAttributes []Attribute) {
+func AddCustomAttributes(metricSet *Set, customAttributes []attributes.Attribute) {
```

Comment:
out of scope:
bad initial design, this should have been method and variadic, now modifying will cause BC